src/Collection/Collection.errors.ts

```diff
@@ -29,6 +29,12 @@ export class AlreadyPublishedCollectionError extends Error {
   }
 }
 
+export class URNAlreadyInUseError extends Error {
+  constructor(public id: string, public urn: string, action: CollectionAction) {
+    super(`The selected URN is already being used. It can be ${action}.`)
```

Comment:
What do you think about this text?
```suggestion
    super(`The provided URN already belongs to a collection. The collection can be ${action}.`)
```

Reply:
That's fine, I believe it should be `the URN provided ...` right?